actionview/test/template/form_collections_helper_test.rb

```diff
@@ -490,7 +490,7 @@ def with_collection_checkboxes(*args, &block)
 
   test "collection check boxes accepts a block to render the label as check box wrapper" do
     with_collection_checkboxes :user, :active, [true, false], :to_s, :to_s do |b|
-      b.label { b.checkbox }
+      b.label { b.checkbox.to_s }
```

Comment:
I think that this might break code in the wild.

I personally write custom view helpers like this (without the `to_s`).